web/src/components/camera/CameraImage.tsx

```diff
@@ -27,18 +27,29 @@ export default function CameraImage({
   const enabled = config ? config.cameras[camera].enabled : "True";
   const [isPortraitImage, setIsPortraitImage] = useState(false);
 
+  const [{ width: containerWidth, height: containerHeight }] =
+    useResizeObserver(containerRef);
+
+  const requestHeight = useMemo(() => {
+    if (!config || containerHeight == 0) {
+      return 360;
+    }
+
+    return Math.min(
+      config.cameras[camera].detect.height,
+      Math.round(containerHeight * 1.1),
+    );
+  }, [config, camera, containerHeight]);
+
   useEffect(() => {
     if (!config || !imgRef.current) {
       return;
     }
 
-    imgRef.current.src = `${apiHost}api/${name}/latest.jpg${
```

Comment:
I can def see a reduction of image quality on my mobile now. Looks worse. I liked the high quality previews before. Anyway we can have this as a configurable option?

Reply:
the bandwidth usage was considerable and it had the possibility of causing live to take longer to startup. It is possible adjustments will be made to the quality but I don't see this being configurable

Reply:
Thanks. The quality looks pretty bad for me and is a big downgrade. I hope that can be addressed